sdk/core/Azure.Core.Amqp/src/AmqpAddress.cs

```diff
@@ -44,7 +44,7 @@ public override string ToString() =>
         /// <see langword="true" /> if the specified object is equal to the current object; otherwise, <see langword="false" />.
         /// </returns>
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public override bool Equals(object obj)
+        public override bool Equals(object? obj)
```

Comment:
Nullable is enabled for this library - https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/core/Azure.Core.Amqp/src/Azure.Core.Amqp.csproj#L9